ui/lib/pki/addon/routes/keys/key/details.js

```diff
@@ -1,3 +1,8 @@
-import Route from '@ember/routing/route';
+import PkiKeysIndexRoute from '.';
 
-export default class PkiKeyDetailsRoute extends Route {}
+export default class PkiKeyDetailsRoute extends PkiKeysIndexRoute {
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+    controller.breadcrumbs.push({ label: this.controller.model.id });
```

Comment:
Could be cleaned up:
```suggestion
    controller.breadcrumbs.push({ label: resolvedModel.id });
```

Reply:
oh duh! thank you 😄